validator/jobroleValidator.ts

```diff
@@ -1,9 +1,25 @@
-import { JobRole } from "../service/model/jobrole";
+import { JobRole } from "../model/jobrole";
 
 module.exports.validateJobrole = function (jobrole: JobRole): string {
     if (jobrole.jobRole.length > 70) {
         return "Job role greater than 50 characters";
     }
 
+    if (jobrole.jobSpecification.length > 70) {
```

Comment:
Similar to other PR, not required for Reading of information. Would be more relavent when it comes to getting user input.